src/components/RichTextInput/RichTextInput.tsx

```diff
@@ -0,0 +1,116 @@
+﻿import React, { useEffect, useState, useRef, useMemo, FC, Suspense } from 'react'
+import classNames from 'classnames';
+import styles from '@/components/RichTextInput/RichTextInput.module.scss'
+
+const DEFAULT_BUTTONS = 'bold,strikethrough,underline,italic,|,ul,ol,brush,paragraph,,image,video,table,link,align,source, fullsize';
+
+type RichTextInputProps = {
+  description?: string;
+  label?: string;
+  value?: string;
+  /**
+   * Default: en
+   */
+  language?: string;
+  onChange?: (newValue: string) => void,
+  tabIndex?: number;
+  className?: string;
+  /**
+   * buttons use for jodit-react
+   * Default: bold,strikethrough,underline,italic,|,ul,ol,brush,paragraph,,image,video,table,link,align,source, fullsize
+   */
+  buttons?: string;
+  minHeight?: number;
+  readonly?: boolean;
+  placeholder?: string;
+  toolbarAdaptive?: boolean;
+  colorPickerDefaultTab?: 'background' | 'color';
+  /**
+   * List of plugins that will be initialized in safe mode.
+   */
+  safePluginsList?: string[];
+  /**
+   * Text when lazy loading jodit-react lib
+   * Default: 
+   */
+  lazyLoadingText?: string;
+}
+
+export const RichTextInput: FC<RichTextInputProps> = (props) => {
+  const { 
+    description, 
+    label, 
+    language = 'en', 
+    onChange, 
+    tabIndex = 0, 
+    className, 
+    buttons = DEFAULT_BUTTONS,
+    minHeight = 140,
+    readonly = false,
+    placeholder = '',
+    toolbarAdaptive = false,
+    colorPickerDefaultTab = 'color',
+    safePluginsList = [],
+    lazyLoadingText = 'Loading...'
+  } = props
+
+  const touched = useRef(false)
+  const editor = useRef(null);
+  const [value, setValue] = useState(props.value || '')
```

Comment:
Please don't keep the value here in the state and make the component uncontrolled. `onChange` and `value` only from the props should do the job here.